tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
A lot of commented code in this file makes it convoluted. Can we keep what we need for now and add to it later?